feat: new variable to control Kafka URP topics retries

erikgb opened the pull request:
# Description

This PR makes it possible to configure the number of retries while waiting for URP topics to stabilize during Kafka health checks.

We are using the health checks tasks when OS-patching our clusters. For some major patches that require a node reboot, a restarted broker needs more time to catch up on replicating partitions. With the current settings, this fails the play, which also wastes a lot of resources on fetching log files for troubleshooting (not required). We think the number of retries while waiting for URP topics to stabilize should be configurable - as it will depend on the cluster load.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [x] Any variable/code changes have been validated to be backwards compatible (doesn't break upgrade)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If required, I have ensured the changes can be discovered by cp-ansible discovery codebase
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

erikgb commented:
Please advice which branch to target for this PR. We are currently using the `v7.7.2` tag, but 7.8.0 seems to be your latest release.

rrbadiani commented:
> Please advice which branch to target for this PR. We are currently using the `v7.7.2` tag, but `7.8.0` seems to be your latest release.

We add new variables to newer .x branches. So 7.9.x would be the correct branch for this PR.

erikgb commented:
> > Please advice which branch to target for this PR. We are currently using the `v7.7.2` tag, but `7.8.0` seems to be your latest release.
> 
> We add new variables to newer .x branches. So 7.9.x would be the correct branch for this PR.

Thanks @rrbadiani! Rebased and switched target branch.

erikgb commented:
@rrbadiani, a gentle ping. I assume this project is still maintained by Confluent? Or?
